test-data/unit/check-semanal-error.test

```diff
@@ -50,10 +50,15 @@ A().x = '' # E: Incompatible types in assignment (expression has type "str", var
 
 [case testInvalidBaseClass2]
 X = 1
-class A(X): # E: Invalid type "__main__.X"
+class A(X): # E
     x = 1
-A().foo(1) # E: "A" has no attribute "foo"
-A().x = '' # E: Incompatible types in assignment (expression has type "str", variable has type "int")
+A().foo(1) # E
```

Comment:
Huh, good catch! I'm a little surprised the test didn't fail on that. From some quick experimentation now, it looks like these bare `# E` comments aren't checked either for presence or absence.

I'll leave them in (and just cut this one which is wrong) for now because that's the style in the rest of this file, but because they aren't checked and can be wrong, I'm inclined to say we should cut them out everywhere we have them and let the error output below speak for itself.